src/neo/Network/P2P/Payloads/Block.cs

```diff
@@ -9,19 +9,61 @@
 
 namespace Neo.Network.P2P.Payloads
 {
+    /// <summary>
+    /// Represents a block.
+    /// </summary>
     public sealed class Block : IEquatable<Block>, IInventory
     {
+        /// <summary>
+        /// The header of the block.
+        /// </summary>
         public Header Header;
+
+        /// <summary>
+        /// The transaction list of the block.
+        /// </summary>
         public Transaction[] Transactions;
 
         public UInt256 Hash => Header.Hash;
+
+        /// <summary>
+        /// The version of the block.
+        /// </summary>
         public uint Version => Header.Version;
+
+        /// <summary>
+        /// The hash of the previous block.
+        /// </summary>
         public UInt256 PrevHash => Header.PrevHash;
+
+        /// <summary>
+        /// The merkle root of the transactions.
+        /// </summary>
         public UInt256 MerkleRoot => Header.MerkleRoot;
+
+        /// <summary>
+        /// The timestamp of the block.
+        /// </summary>
         public ulong Timestamp => Header.Timestamp;
+
+        /// <summary>
+        /// The index of the block.
+        /// </summary>
         public uint Index => Header.Index;
+
+        /// <summary>
+        /// The primary index of the consensus nodes that generates the block.
```

Comment:
`of the consensus node that generated this block`.

Reply:
Updated.